tomoni/style.css

```diff
@@ -5,13 +5,13 @@ Author: Automattic
 Author URI: https://automattic.com/
 Description: Tomoni is a theme that displays a use case of multilingual content with Japanese.
 Requires at least: 5.8
```

Comment:
Since it uses `css` in theme.json, we should make min version as `6.3`?

Reply:
I'll update this, too. Thank you for pointing this out!